src/wallets/EthersWallet.ts

```diff
@@ -76,6 +76,18 @@ export class EthersWallet implements TLWallet {
     }
   }
 
+  /**
+   * Deploys a new identity contract on the chain
+   */
+  public async deployIdentity(): Promise<string> {
+    // This does not have to deploy any identity, because it does not have it
+    return this.address
+  }
+
+  public async isIdentityDeployed(): Promise<boolean> {
```

Comment:
should we not rather return true? 
in a app flow it would probably do something like 
```
if not isIdentityDeploy
   deployIdentity()
```

so if it uses no identity wallets it should maybe just return true, as in all is good?

Reply:
Seems abusive, but sure.